include/tcpdf_fonts.php

```diff
@@ -1787,7 +1787,7 @@ public static function UTF8ArrayToUniArray($ta, $isunicode=true) {
 
 	/**
 	 * Extract a slice of the $strarr array and return it as string.
-	 * @param string $strarr The input array of characters.
+	 * @param array $strarr The input array of characters.
```

Comment:
```suggestion
	 * @param string[] $strarr The input array of characters.
```
Would it match better ?